Iteration3/master/counter.py

```diff
@@ -0,0 +1,59 @@
+# https://github.com/jd/fastcounter
+import itertools
+import threading
+
+
+class Counter:
+    __slots__ = (
+        "value",
+        "_step",
+    )
+
+    def __init__(self, init=0, step=1):
+        self.value = init
+        self._step = step
+
+    def increment(self, num_steps=1):
+        self.value += self._step * num_steps
+
+
+class FastReadCounter(Counter):
```

Comment:
Why do we need it, if it's not used anywhere?

Reply:
It is like default part of foreign class in case of refactoring it could be deleted

Reply:
resolved in c78054bc0a202ffb1e889cd723865e480abdcffc by @popenyuk